rpc/rpcserver/server.go

```diff
@@ -2750,3 +2750,27 @@ func (s *decodeMessageServer) DecodeRawTransaction(ctx context.Context, req *pb.
 
 	return resp, nil
 }
+
+func (s *walletServer) BestBlock(ctx context.Context, req *pb.BestBlockRequest) (
+	*pb.BestBlockResponse, error) {
+	n, err := s.requireNetworkBackend()
+	if err != nil {
+		return nil, err
+	}
+
+	chainClient, err := chain.RPCClientFromBackend(n)
+	if err != nil {
+		return nil, translateError(err)
+	}
+
+	hash, height, err := chainClient.GetBestBlock()
+	if err != nil {
+		return nil, translateError(err)
+	}
+
+	resp := &pb.BestBlockResponse{
+		Hash:   hash.CloneBytes(),
```

Comment:
a copy/clone is unnecessary

Reply:
Elaborate, `hashCloneBytes()` is equivalent to `[]bytes(hash.String())`, or should I be returning the hash string instead?

Reply:
`hash[:]`